.github/workflows/debos.yml

```diff
@@ -0,0 +1,86 @@
+name: Build debos recipe
+
+on:
+  # run on pull requests to the main branch if debos recipes are changed
+  pull_request:
+    branches: [main]
+    paths: ['debos-recipes/*']
+  # run on pushes if debos recipes are changed
+  push:
+    branches: [main]
+    paths: ['debos-recipes/*']
+  # run daily at 8:30am
+  schedule:
+    - cron: '30 8 * * *'
+  # allow manual runs
+  workflow_dispatch:
+
+# only need permission to read repository; implicitely set all other
+# permissions to none
+permissions:
```

Comment:
why do you need that?

Reply:
By default, everything is allowed because potentially workflows could be used for anything, but we know what the workflow will do